build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.24.0-go1.18.9-bullseye.0
+v1.25.0-go1.19.4-bullseye.0
```

Comment:
you can replace with `v1.24.0-go1.19.4-bullseye.0` now :)

Reply:
done